mypy/report.py

```diff
@@ -304,7 +304,7 @@ def __init__(self, reports: Reports, output_dir: str) -> None:
         self.css_html_path = os.path.join(reports.data_dir, 'xml', 'mypy-html.css')
         xsd_path = os.path.join(reports.data_dir, 'xml', 'mypy.xsd')
         self.schema = etree.XMLSchema(etree.parse(xsd_path))
-        self.last_xml = None  # type: etree._ElementTree
+        self.last_xml = None  # type: Optional[etree._ElementTree]
```

Comment:
I wonder if it wouldn't be better to keep this as non-optional and remove the None assignments? Not sure I follow the logic there.